components/health_controller/src/reporters.rs

```diff
@@ -92,17 +135,85 @@ impl UnifiedSlowScore {
         &mut self.factors[factor as usize]
     }
 
-    // Returns the maximum score of all factors.
-    pub fn get_score(&self) -> f64 {
-        self.factors
-            .iter()
-            .map(|factor| factor.get())
-            .fold(1.0, f64::max)
+    // Returns the maximum score of disk factors.
+    pub fn get_disk_score(&mut self) -> f64 {
+        (self.factors[InspectFactor::RaftDisk as usize].get()
+            + self.factors[InspectFactor::KvDisk as usize].get())
+            / 2.0
+    }
+
+    pub fn get_network_score(&self) -> HashMap<u64, u64> {
+        self.network_factors
+            .lock()
+            .unwrap()
+            .iter_mut()
+            .map(|(k, v)| (*k, v.get() as u64))
+            .collect()
     }
 
     pub fn last_tick_finished(&self) -> bool {
         self.factors.iter().all(SlowScore::last_tick_finished)
     }
+
+    pub fn record_timeout(&mut self, factor: InspectFactor) {
```

Comment:
Please add necessary annotations here to explain the reason why it does not take `Network` into account